projects/step-frontend/src/lib/modules/grid/components/status-summary/status-summary.component.html

```diff
@@ -0,0 +1,39 @@
+<div
+  class="progress"
+  matTooltip="Used: {{ tokenGroup.countByState!['IN_USE'] }}, In error: {{
+    tokenGroup.countByState!['ERROR']
+  }}, Total: {{ tokenGroup.capacity }} "
+>
+  <div
+    class="progress-bar token-status-IN_USE"
+    [style]="'color:white; width:' + (tokenGroup.countByState!['IN_USE']! / tokenGroup.capacity!) * 100 + '%;'"
```

Comment:
I would suggest to `color: white` to css class.
Then create additional pipe, that will contain the percentage calculation logic for token group.
A result binding will look like this:
```
<div class="progress-bar token-status-IN_USE [style.width.%]="tokenGroup | tokenGroupWidth: 'IN_USE'">
....
</div>
<div class="progress-bar token-status-ERROR [style.width.%]="tokenGroup | tokenGroupWidth: 'ERROR'">
....
</div>
```

Reply:
We could also create a dedicated component for the progress bar, I think we use it in multiple views. That would make it much more readible.

Reply:
I have created a dedicated component for the progress bar, but for now it is not exported from the grid module. we could leave it like that for now and if we need it in other modules we could move it to `_common/components`

Reply:
I checked in which views the legacy directive `<grid-status-distribution>` is used, and it is only used in GridAgents and GridTokenGroup. So keeping the new counterpart-component private in the `grid.module` should be fine